api/policies/v1/policygroup_validation.go

```diff
@@ -44,6 +62,12 @@ func validatePolicyGroupMembers(policy Policy) *field.Error {
 	if len(policy.GetPolicyGroupMembers()) == 0 {
 		return field.Required(field.NewPath("spec").Child("policies"), "policy groups must have at least one policy member")
 	}
+	for memberName := range policy.GetPolicyGroupMembers() {
+		_, matchReservedSymbol := celReservedSymbols[memberName]
+		if len(memberName) == 0 || matchReservedSymbol || !idenRegex.MatchString(memberName) {
+			return field.Invalid(field.NewPath("spec").Child("policies"), memberName, "policy group member name is invalid")
```

Comment:
this function should now return `field.ErrorList` and we should append here instead of returning